Ejercicio 16 GIT

ijchavez opened the pull request:
Se agrega el ejercicio 16 al git. Adicionalmente se usa una clase llamada seleniumUtils con el siguiente contenido:

package Clase9.Ejercicio;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class seleniumUtils {
    public static WebDriver getUrlDriver(String unaURL) {
        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
        WebDriver driver = new ChromeDriver();
        //driver = new ChromeDriver();
        driver.get(unaURL);

        return driver;
    }
}

ijchavez committed:
Ejercicio 16 GIT 

Se agrega el ejercicio 16 al git. Adicionalmente se usa una clase llamada seleniumUtils con el siguiente contenido:

package Clase9.Ejercicio;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class seleniumUtils {
    public static WebDriver getUrlDriver(String unaURL) {
        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
        WebDriver driver = new ChromeDriver();
        //driver = new ChromeDriver();
        driver.get(unaURL);

        return driver;
    }
}

ijchavez commented:
Hola Emiliano, mil disculpas, abri el pull request y no agregue a seleniumcurso como revisor

seleniumcurso commented on Practico9Ejercicio16/Ejercicio16.java:
```diff
@@ -0,0 +1,83 @@
+        //y buscando el boton puse copy >>xpath, y le pegue lo que me devolvio.
+        driver.findElement(By.xpath("//*[@id=\"appMountPoint\"]/div/div/div/div/div/div[1]/div/a")).click();
+
+        List<WebElement> elemH1 = driver.findElements(By.tagName("h1"));
```
mejorar el nombre de las listas

ijchavez replied:
Hecho

ijchavez committed:
Correcciones

- Se corrigieron los nombres de las listas
- Se agrego el nombre del elemento que mas se repite

ijchavez reviewed (commented):
No se porque no me aparecia actualizado este ejercicio en GIT. Lo actualice con los nombres mas descriptivos y el String de listas para ver cual es el elemento mas repetido
